docs/source/index.rst

```diff
@@ -56,9 +67,10 @@ information here will make it easier for newcomers to get started using
 
 
 
-Navigation
-==================
+How to cite Manim
```

Comment:
Any reason for adding this heading? You had already mention at the top on where to refer so I think we can remove this.

Reply:
Commited by mistake, fixed via 3f5608ac